int32 ControlMode
float32 MotorPosition
float32 MotorVelocity
